src/app/information/information.component.sass

```diff
@@ -0,0 +1,44 @@
+.information_wrapper
```

Comment:
class selector must named without underscore and camelcase
use BEM symbols like a `-, __, --`

Reply:
Some descriptions for this, BEM - Block Element Modificator, in this case block name contains two words "information" and "wrapper", so correct classname is `.information-wrapper`. Notice using sign '-' here. 

If you will have nested elements under this block, for example:
```
.information-wrapper
  .information-wrapper__title
  .information-wrapper__description
```
then you should use double lodash to mark this classname as Element, according to BEM notation. 

If you will have couple of variants for title, you can use Modificator notation, for example:
```
.information-wrapper
  .information-wrapper__title--dashed
  .information-wrapper__description--dashed
```
```
.information-wrapper
  .information-wrapper__title--marquee
  .information-wrapper__description--marquee
```
```
.information-wrapper
  .information-wrapper__title--some-other-decoration-example
  .information-wrapper__description--some-other-decoration-example
```